### ISSUE TITLE
Alexa Media TTS support

### ISSUE THREAD

**Issue by alandtse** [kalkih/mini-media-player#96]
Hi,
For the [alexa_media_player](https://github.com/keatontaylor/alexa_media_player) custom component we're about to deprecate our `media_player.alexa_tts` service in the next version or so `1.2.0` and I wanted to give you a heads up since you support that service call for our users.

Instead of the old service, the new default service should be: `notify.alexa_media`.  The json data will be
```json
{
"message":"test",
"data":{"type":"tts"},
"target":["media_player.kitchen"]
}
```
We'll likely deprecate the service after one full release cycle to confirm parity between `notify` and `media_player.alexa_tts`.

**Comment by kalkih** [comment]
Hey,
Thanks for the heads up.
Just to be sure, does the target parameter support an array of more than one entity?

Thanks.

**Comment by kalkih** [comment]
@alandtse Another question, is string accepted as target value, or does it have to be an array, even if there's only one target?

**Comment by kalkih** [comment]
It seems like it does, from a quick peek at the source.
https://github.com/keatontaylor/alexa_media_player/blob/4b9701f61023b3252e915f0295b2b157705fe188/alexa_media/notify.py#L63-L65

**Comment by alandtse** [comment]
Yes. It will support both a string of an array of strings. FYI, we released 1.2.0 and will keep the old service till 1.3.0.

**Comment by alandtse** [comment]
Closing as I believe this is resolved. Thanks again!
